rosalind_hamm.py

```diff
@@ -1,3 +1,3 @@
 def main(data):
     a, b = data.strip().split('\n')
-    return sum([1 for i in range(len(a)) if a[i] != b[i]])
+    return sum([l[0] != l[1] for l in zip(a, b)])
```

Comment:
This is correct. You could do one last thing to make it a little more readable, and that's using argument unpacking for `l[0]` and `l[1]`.

```py
return sum([(aa != bb) for aa, bb in zip(a, b)])
```

Nice work!